transaction/core/src/blockchain/block_version.rs

```diff
@@ -91,6 +91,11 @@ impl BlockVersion {
     pub fn validate_transaction_outputs_are_sorted(&self) -> bool {
         self.0 > 3
     }
+
+    /// mint transactions are introduced in block version 3
```

Comment:
ideally there would be a link to an MCIP that explains what this is

Reply:
the mcip hasn't been merged yet

Reply:
does the MCIP exist anywhere? even if it's a placeholder, i think it is helpful for these "is X supported" functions to have a link in the code comment to the description of what feature X is

Reply:
I dont believe there's an MCIP yet. @sugargoat I think you might've been the point of contact on that?